azureDatabaseUtils/src/test/java/bio/terra/workspace/azureDatabaseUtils/database/DatabaseServiceTest.java

```diff
@@ -14,6 +18,7 @@ public class DatabaseServiceTest extends BaseUnitTest {
 
   @MockBean private DatabaseDao databaseDao;
   @MockBean private Validator validator;
+  @MockBean private BlobStorage storage;
```

Comment:
are you streaming to a storage container in azure when runing these tests?

Reply:
no - these are unit tests - but that's a good reminder that I should have some kind of connected test here too

Reply:
now I think it might be better to test this in the context of the stairway steps that call these utilities, which will come in a later PR

Reply:
got it, my suggestion was going to be to use a mock for this, something like local file storage (so you could test that whatever is written is what you expect) without needing to connect to blob storage/etc